app/views/groups/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<section class="bg-light">
+  <div class="d-flex border container gap-5 align-items-center">
+    <%= link_to '/categories', class: 'text-light h1' do %>
+      <i class="bi bi-arrow-left"></i>
+    <% end %>
+    <h2 class="text-light">DETAILS</h2>
+  </div>
+  <div class="d-flex justify-content-between align-items-center container shadow-sm bg-light ">
+    <h5>Total Amount:</h5>
+    <h5><%= number_to_currency(@group_expenses.sum(:amount)) %></h5>
+  
+  </div>
+  <div class="expenses m-3">
```

Comment:
- [OPTIONAL] Kindly follow the provided design. The card should have a category image, name, `created date`, and amount. kindly let display the creation date for each category card. 🛀🏻